docs/tutorials.rst

```diff
@@ -15,22 +15,28 @@
 Tutorials
 =========
 
-The following tutorials demonstrate how to use Trieste to solve various optimization problems, and detail the functionality available in Trieste.
+Example optimization problems
```

Comment:
this is the first title you wanted to change

Reply:
I actually can't find anything better